roles/swift-common/tasks/main.yml

```diff
@@ -76,6 +76,11 @@
     - logging
   when: logging.enabled|default('True')|bool
 
+- name: change swift log file permissions
+  file: path={{ item }} mode=0644 owner=swift group=swift state=touch
```

Comment:
the swift logging directory is being created [here](https://github.com/blueboxgroup/ursula/blob/master/roles/swift-common/tasks/logging.yml#L2-L3) owned by syslog:adm which allows syslog to write to the logfiles ( which would inherit the perms from the dir to be 640 [i think] to syslog:adm as they get created by syslog ).

The perms listed here would mean that syslog would not be able to log on behalf of swift which would likely then break swift.